recipes/stripe.rb

```diff
@@ -117,6 +117,12 @@
   action :create
 end
 
+# Make sure that the permissions for the  'mysql' directory are set correctly.
+# When recovering from a backup uids could have changed.
+execute "change permissions #{new_mysql_dir} owner" do
+  command "chown --recursive --silent mysql:mysql #{new_mysql_dir}"
```

Comment:
There should be a guard here so prevent this from running when it does not have to.